worker/test/src/RTC/TestRtpPacket.cpp

```diff
@@ -127,7 +129,7 @@ SCENARIO("parse RTP packets", "[parser][rtp]")
 		REQUIRE(packet->ReadAbsSendTime(absSendTime) == true);
 		REQUIRE(absSendTime == 0x65341e);
 
-		auto* clonedPacket = packet->Clone();
+		auto clonedPacket = packet->Clone();
```

Comment:
Why not auto* anymore?

Reply:
before this was representing a raw pointer, now it's a shared pointer instance. I believe is makes more sense since the shared pointer itself is not a pointer but it contains one.